packages/py3_setuptools.rb

```diff
@@ -3,22 +3,27 @@
 class Py3_setuptools < Python
   description 'Setuptools is the python build system from the Python Packaging Authority.'
   homepage 'https://setuptools.readthedocs.io/'
-  version '72.1.0-py3.12'
+  version '74.1.2-py3.12'
   license 'MIT'
   compatibility 'all'
   source_url 'https://github.com/pypa/setuptools.git'
   git_hashtag "v#{version.split('-').first}"
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '5365ac36cf205911283005414f50d2d1d100a527678b3343058abda6675cfff9',
-     armv7l: '5365ac36cf205911283005414f50d2d1d100a527678b3343058abda6675cfff9',
-       i686: 'd24f343a04d8cefdcb132c38c1d5cbee3992f72416eff1a4a5f9b5eb2e482c34',
-     x86_64: '416b19dfef5ab021a7248fe33f27082275e228ce8a0321227bd80695302d4e1f'
+    aarch64: 'f3a0f31ba3c061d079764be008239b14e40ee4b8cac60e0f75d39d7f9d5e0e2f',
+     armv7l: 'f3a0f31ba3c061d079764be008239b14e40ee4b8cac60e0f75d39d7f9d5e0e2f',
+       i686: '49d57dcd49c10f827a328a744ad31f3573b1b015f2e763840b91ef2091b12456',
+     x86_64: 'b9dc353616f26bad1143e3c57ab7470ad16cd8ae400465363ef2132aaf44eaee'
   })
 
-  depends_on 'python3', '< 3.13.0'
+  depends_on 'python3'
   depends_on 'py3_packaging'
 
   conflicts_ok
+
+  def self.prebuild
+    system 'python3 -m pip uninstall setuptools -y'
```

Comment:
Why don't we just convert this to a pip buildsystem?

Reply:
The rub here is pip setuptools is needed to build this package.

Reply:
ah